extensions/mongodb/src/main/java/com/hazelcast/jet/mongodb/impl/MongoUtilities.java

```diff
@@ -140,4 +144,23 @@ public static LocalDateTime bsonTimestampToLocalDateTime(@Nullable BsonTimestamp
                             .withZoneSameInstant(systemDefault())
                             .toLocalDateTime();
     }
+
+    static void checkCollectionExists(MongoDatabase database, String collectionName) {
```

Comment:
checkXXXOrFail kind of method names are easier to understand that it throws exception. It is up to you

Reply:
maybe `assertCollectionExists`

Reply:
maybe `assertCollectionExists`

Reply:
I think it's a common thing that `checkX` methods fail. `assert` is good also, but more test-related IMHO